### DISCUSSION TITLE
user changing password.

### DISCUSSION THREAD

**Discussion by applemuncy** [ltsp/ltsp#384] [Q&A] [answered]
I want to create users with a temporary password and have the user change it.  
Using passwd on the client doesn't propagate back to the server.  Is this a problem with pamltsp?
Or am I doing something wrong?

**Comment by alkisg** [comment, marked as answer]
It's by design, see https://github.com/ltsp/ltsp/issues/148#issuecomment-683382804

To work around it while using pamltsp, you'd need to create your own wrapper application to handle password updates,
or, for a more proper solution, you'd use LDAP/SSSD: https://github.com/ltsp/ltsp/issues/338
